src/libmongoc/src/mongoc/mongoc-change-stream.c

```diff
@@ -243,6 +243,10 @@ _make_cursor (mongoc_change_stream_t *stream)
    bson_init (&command);
    bson_copy_to (&(stream->opts.extra), &command_opts);
 
+   if (stream->opts.comment.value_type != BSON_TYPE_EOD) {
+      bson_append_value (&command_opts, "comment", 7, &stream->opts.comment);
+   }
```

Comment:
This is not currently tested as the change stream spec tests are currently skipped. More work is required to update the unified test runner ([CDRIVER-3973](https://jira.mongodb.org/browse/CDRIVER-3973)) and that seemed like too much for this PR.

I'd propose scheduling those tickets immediately after this work (or at least in the same fixVersion).